server/common-functions.js

```diff
@@ -1,22 +1,26 @@
 const rp = require('request-promise');
```

Comment:
@Shapokhan can we remove this and also uninstall `request-promise` from our project if we have replaced all of its references with `axios` ?